code/src/reducer/todos.js

```diff
@@ -0,0 +1,64 @@
+import {createSlice} from '@reduxjs/toolkit'
+
+export const todos = createSlice({
+    name:'todos',
+    initialState: {
+        username: 'Bam',
+        items: [
+            {
+                id: 1, 
+                text: 'Add your first task',
+                category: 'task',
+                isComplete: false,
+                created: 1606387233993
+            },
+            {
+                id: 2, 
+                text: 'Walk the dog',
+                category: 'task',
+                isComplete: false,
+                created: 1606387233993
+            },
+            {
+                id: 3, 
+                text: 'Treat yourself',
+                category: 'task',
+                isComplete: false,
+                created: 1606387233993
+            },
+        ]
+    }, 
+    
+    reducers: {
+        removeOne: (store, action) => {
+            store.items = store.items.filter((item) => item.id !== action.payload)
+        },
+
+        addTask: (store, action) => {
+            const newTask = {
+                id: Math.max(...store.items.map(item => item.id)) + 1,
```

Comment:
I really like that you used Math.max for the id! 👍 I used uniq id myself, recommend you to try that as well sometime!